app/abilities/base_ability.rb

```diff
@@ -0,0 +1,6 @@
+class BaseAbility
+  include CanCan::Ability
+
+  def initialize(user)
+  end
```

Comment:
moreover - the include at the top helps make it more symmetrical. Without that, I'd say def put the line of WS in.